### PR TITLE
Added Patient basis field for base configuration

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-5536](https://jira.cms.gov/browse/MAT-5536)
(Optional) Related Tickets:

### Summary

image

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx [hunk 1]
```diff
@@ -200,7 +200,31 @@ const BaseConfiguration = () => {
               })}
             />
           </div>
-          <div className="right"></div>
+          <div className="right">
+            <RadioButton
+              row
+              id="patient-basis"
+              dataTestId="patient-basis"
+              label="Patient Basis:"
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx]: I think no need of colons here

Comment (RohitKandimalla) [reply src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx]: ooh yeah you are right, I will remove them

File: src/validations/QDMMeasureSchemaValidator.ts [hunk 2]
```diff
@@ -13,4 +13,5 @@ const QDMBaseConfigurationTypesSchema = Yup.array().min(
 export const QDMMeasureSchemaValidator = Yup.object().shape({
   scoring: QDMMeasureScoringSchema,
   baseConfigurationTypes: QDMBaseConfigurationTypesSchema,
+  patientBasis: Yup.boolean().required("Patient Basis field is required"),
```
Comment (adongare) [inline src/validations/QDMMeasureSchemaValidator.ts]: We probably do not need the word "field" in the message but if it is a requirement ignore this.

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx [hunk 3]
```diff
@@ -19,12 +19,14 @@ import {
   Select,
   MadieDiscardDialog,
   Toast,
+  RadioButton,
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx:22]: I feel this should have been named RadioGroup or something similar. We can refactor some other time.

**Review by adongare — DISMISSED** [review]

File: src/validations/QDMMeasureSchemaValidator.ts (hunk 2 above)
Comment (RohitKandimalla) [reply src/validations/QDMMeasureSchemaValidator.ts]: Technically user will never see this error message as there is a default value and the radio button cannot be unchecked, Story doesn't specify any message, So I will update it.

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.scss [hunk 4]
```diff
@@ -16,13 +16,13 @@
...
-      }
-    }
+    //> .base-configuration-container {
+    //  display: flex;
+    //  flex-direction: column;
+    //  align-items: flex-start;
+    //  > label {
+    //    font-family: Rubik, sans-serif;
+    //  }
+    //}
```
Comment (sb-cecilialiu) [inline src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.scss:26]: Are these commented out needed?

**Pull request closed without merging**